webui/react/src/pages/ExperimentDetails/ExperimentTrials.tsx

```diff
@@ -166,6 +166,21 @@ const ExperimentTrials: React.FC<Props> = ({ experiment, pageRef }: Props) => {
       );
     };
 
+    const logRetentionDaysRenderer = (_: string, record: TrialItem): React.ReactNode => {
+      const logRetentionDays = record.logRetentionDays; //  to take advantage of ts type
+      if (logRetentionDays === undefined) {
+        // only `logRetentionDays` causes unwanted behavior when the value is 0
+        return <span>-</span>; // or null depends of what you wanna show
```

Comment:
nit: you can remove these comments. it was for explanations.

Reply:
done